main.py

```diff
@@ -527,11 +537,61 @@ async def missions_history():
 
         return JSONResponse(content=response.json())
     except requests.RequestException as e:
-        raise HTTPException(status_code=500, detail=f"Error fetching missions history: {str(e)}")
+        raise HTTPException(
+            status_code=500, detail=f"Error fetching missions history: {str(e)}"
+        )
+
+
+@app.get("/v1/screenshot")
```

Comment:
@sanatem Add everything new in v1 to ensure that the other endpoints keep working, and the researchers can continue using their model.